src/workloads/docs/StreamStatsReporter.yml

```diff
@@ -29,7 +30,9 @@ Actors:
       OperationName: RunCommand
       OperationCommand:
         streams_startStreamProcessor: ""
+        tenantId: *TenantId
         name: *StreamProcessorName
+        processorId: *StreamProcessorName
```

Comment:
As noted above `processorId` is never actually used in this code. Either remove it or change the cpp to get it.